.env.example

```diff
@@ -2,17 +2,15 @@
 ## Then get your OpenAI API Key here: https://platform.openai.com/account/api-keys
 OPENAI_API_KEY=XXXXXXXX
 
-## Generate a random secret: https://generate-secret.vercel.app/32
-NEXTAUTH_SECRET=XXXXXXXX
-
-## Only required for localhost
-NEXTAUTH_URL=http://localhost:3000
-
+## Generate a random secret: https://generate-secret.vercel.app/32 or `openssl rand -base64 32`
+AUTH_SECRET=XXXXXXXX
 ## Create a GitHub OAuth app here: https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app
 AUTH_GITHUB_ID=XXXXXXXX
 AUTH_GITHUB_SECRET=XXXXXXXX
+## Support OAuth login on preview deployments, see: https://authjs.dev/guides/basics/deployment#securing-a-preview-deployment
+AUTH_REDIRECT_PROXY_URL=https://auth.example.com/api/auth
```

Comment:
Update readme with instructions? What if this isn't set? This will fail when running locally before deployment.

Reply:
Yeah, I can update the README. Regarding the environment variable, IMO one should not use the same github app for production and development anyway, so I would not set this variable locally.

Reply:
So then let's both add this information to readme and in this comment